runtime/Makefile

```diff
@@ -47,9 +47,12 @@ integ-test:
 			--privileged \
 			--ipc=host \
 			--volume /dev:/dev \
-			--volume /sys:/sys \
 			--volume /run/udev/control:/run/udev/control \
 			--volume $(CURDIR)/logs:/var/log/firecracker-containerd-test \
+			--device=/dev/vhost-vsock:/dev/vhost-vsock \
```

Comment:
We are bind-mounting /dev from the host, so I didn't think we'd need to explicitly declare any of these devices